backend/connector-integration/src/connectors/nexinets.rs

```
@@ -0,0 +1,566 @@
+    http_method: Post,
+    other_functions: {
+        fn get_headers(
+            &self,
+            req: &RouterDataV2<Capture, PaymentFlowData, PaymentsCaptureData, PaymentsResponseData>,
+        ) -> CustomResult<Vec<(String, Maskable<String>)>, errors::ConnectorError> {
+            self.build_headers(req)
+        }
+        fn get_url(
+            &self,
+            req: &RouterDataV2<Capture, PaymentFlowData, PaymentsCaptureData, PaymentsResponseData>,
+        ) -> CustomResult<String, errors::ConnectorError> {
+        let order_id = &req.resource_common_data.connector_request_reference_id;
+        let transaction_id = req.request.connector_transaction_id.get_connector_transaction_id().unwrap_or_else(|_| "missing-transaction-id".to_string());
+        Ok(format!(
+            "{}/orders/{order_id}/transactions/{transaction_id}/capture",
+            self.connector_base_url_payments(req),
+        ))
+        }
+    }
+);
+
+macros::macro_connector_implementation!(
+    connector_default_implementations: [get_content_type, get_error_response_v2],
+    connector: Nexinets,
+    curl_request: Json(NexinetsRefundRequest),
+    curl_response: NexinetsRefundResponse,
+    flow_name: Refund,
+    resource_common_data: RefundFlowData,
+    flow_request: RefundsData,
+    flow_response: RefundsResponseData,
+    http_method: Post,
+    other_functions: {
+        fn get_headers(
+            &self,
+            req: &RouterDataV2<Refund, RefundFlowData, RefundsData, RefundsResponseData>,
+        ) -> CustomResult<Vec<(String, Maskable<String>)>, errors::ConnectorError> {
+            self.build_headers(req)
+        }
+        fn get_url(
+            &self,
+            req: &RouterDataV2<Refund, RefundFlowData, RefundsData, RefundsResponseData>,
+        ) -> CustomResult<String, errors::ConnectorError> {
+           let connector_metadata = req
+    .request
+    .get_connector_metadata()
+    .map_err(|_| errors::ConnectorError::MissingRequiredField {
```

Don't discard the error log the error